OVERVIEW.md

```diff
@@ -221,12 +221,19 @@ Extends defined rules with preset rules.
 
 Type: `String`
 
-Values: `"airbnb"`, `"crockford"`, `"google"`, `"jquery"`, `"mdcs"`, `"node-style-guide"`, `"wikimedia"`, `"wordpress"`, `"yandex"`
+Values: You can choose one of the default presets: `"airbnb"`, `"crockford"`, `"google"`, `"jquery"`, `"mdcs"`, `"node-style-guide"`, `"wikimedia"`, `"wordpress"`, `"yandex"`.
+
+Or you can load from it local path or as node module
 
 #### Example
 
 ```js
 "preset": "jquery"
+"preset": "./path-to-your-preset"
+
+// If your preset called "jscs-your-preset-node_modules-path" 
+// You can either define full name or omit "jscs-" prefix -
```

Comment:
I don't really see the point to the sugar of `jscs-`. It only helps users who want to fork the built-in presets and retain the name. But if they're forking it, they can name it whatever they'd like and just supply that as the preset value.

Any big wins with this prefix?